planning/behavior_path_planner_common/src/utils/drivable_area_expansion/static_drivable_area.cpp

```diff
@@ -1435,20 +1576,35 @@ std::vector<geometry_msgs::msg::Point> calcBound(
   };
 
   // Step1. create drivable bound from drivable lanes.
-  std::vector<lanelet::ConstPoint3d> bound_points = convert_to_points(drivable_lanes);
+  auto [bound_points, skip_post_process] = [&]() {
+    if (!enable_expanding_freespace_areas) {
+      return std::make_pair(convert_to_points(drivable_lanes, is_left), false);
+    }
+    return getBoundWithFreeSpaceAreas(
+      convert_to_points(drivable_lanes, is_left), convert_to_points(drivable_lanes, !is_left),
+      planner_data, is_left);
+  }();
+
+  const auto post_process = [&](const auto & bound, const auto skip) {
+    return skip
+             ? bound
+             : postProcess(
+                 bound, path, planner_data, drivable_lanes, enable_expanding_hatched_road_markings,
+                 enable_expanding_intersection_areas, is_left, is_driving_forward);
+  };
```

Comment:
this is just wrapper of `postProcess` function with skip. seems not need make this lambda